tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/firebase/FirebaseModule.java

```diff
@@ -2,11 +2,13 @@
 
 import com.google.inject.AbstractModule;
 import com.linagora.tmail.james.jmap.method.FirebaseCapabilitiesModule;
+import com.linagora.tmail.james.jmap.method.FirebaseSubscriptionGetMethodModule;
 
 public class FirebaseModule extends AbstractModule {
     @Override
     protected void configure() {
         install(new FirebaseCapabilitiesModule());
-        // JMAP Firebase method binding ...
+        install(new FirebaseSubscriptionGetMethodModule());
+        install(new MemoryFirebaseSubscriptionModule());
```

Comment:
Please remove this memory module. This is not a common module (for both memory and distributed TMail), this variant would be taken care of by the module chooser instead.

see https://github.com/linagora/tmail-backend/pull/517/commits/b3da7455926254895c652c5f7c433dd228c36600

Reply:
if it is removed now, ci will fail. 
or waiting until #517 has been merged, then I will rebase later

Reply:
Merged. Please rebase.